mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/logging/impl/Log.java

```diff
@@ -9,35 +9,42 @@
 
 import java.lang.annotation.Annotation;
 
-import org.hibernate.search.v6poc.entity.pojo.bridge.declaration.BridgeMapping;
 import org.hibernate.search.v6poc.util.SearchException;
 
 import org.jboss.logging.BasicLogger;
-import org.jboss.logging.annotations.Cause;
 import org.jboss.logging.annotations.Message;
 import org.jboss.logging.annotations.MessageLogger;
 
 @MessageLogger(projectCode = "HSEARCH-POJO")
 public interface Log extends BasicLogger {
 
-	@Message(id = 1, value = "Unable to create annotation for definition of type %1$s")
-	SearchException unableToCreateAnnotationForDefinition(Class<? extends Annotation> annotationType, @Cause Exception e);
```

Comment:
It's really not worth it to renumber the other methods when removing one. It's not a big deal if there are a few gaps.